### ISSUE TITLE
Error on creating the integration

### ISSUE THREAD

**Issue by ggonmar** [Daanoz/ha-google-photos#32]
When trying to set up the integration on my HA, I search under "Add Integration" for "Google Photos", I inputted my oauth credentials, and then a popup appears from google saying that there was a Error 400: redirect_uri_mismatch.

![image](https://github.com/Daanoz/ha-google-photos/assets/11583288/d4a84dba-0d2a-4cc3-997d-d4d648416707)

Am I doing something wrong?

Thank you!

**Comment by Daanoz** [comment]
Could you check if your oAuth client looks something like this (especially Authorized redirect urls): 
![Credential page](/Daanoz/ha-google-photos/raw/main/docs/Credentials.png)

Note that you must use the `my.home-assistant.io` url.

**Comment by mmikee02** [comment]
Hello, 
I had the same thing, you should verify if you didn't had entered wrongs credentials previously.
here : https://my.home-assistant.io/redirect/application_credentials/  (it's the link written in the FAQ).
You will see all your different credentials  on the page, if you see 1 "google photo", delete it and re-start the integration of the plugin.
Hope this will help you.
-->Mike

**Comment by Zeunas** [comment]
same is happening to me

**Comment by Daanoz** [comment]
@Zeunas You have the correct redirect_uri set to `my.home-assistant.io`?

**Comment by Zeunas** [comment]
I do, I actually tried the credentials on another HA instance and that one is working. Both instances are on the same version of HAOS, the only difference is that one is on a Rasp pi3 and the other is on a Rasp pi4, the integration is working on the latter.

**Comment by Zeunas** [comment]
I did that a couple of times already but got the same results. One thing I didn't say actually, was that this integration was initially set up on the pi4, then I decided to move to the pi3. I tried using the same credentials that I used for the pi4 and it didn't work, then I deleted the OAuth 2.0 Client ID and recreated again following the instructions on here, that didn't work either (this new Client ID is the one that still works on the pi4). So not sure if I compromised the authentication of pi3 somehow by using pi4 initial in the first place.
